src/index.html

```diff
@@ -5,9 +5,88 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="Apple laptop"
+        >
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p class="card__code">
+            Product code: 195434
+        </p>
+
+        <div class="stars">
+          <div class="stars__icon">
+            <div
+              class="stars__review"
+            >
+            </div>
+            <div
+              class="stars__review"
+            >
+            </div>
```

Comment:
```suggestion
            <div class="stars__review"></div>
            <div class="stars__review"></div>
```

no need to move only one attribute to separate line

Reply:
thanks :)